src/main/java/com/qaprosoft/carina/demo/guipractice/pages/HomePage.java

```diff
@@ -0,0 +1,43 @@
+package com.qaprosoft.carina.demo.guipractice.pages;
+
+import java.util.List;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import com.qaprosoft.carina.core.foundation.utils.Configuration;
+import com.qaprosoft.carina.core.foundation.utils.R;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import com.qaprosoft.carina.demo.guipractice.components.NavigateMenu;
+import com.qaprosoft.carina.demo.guipractice.components.Product;
+
+public class HomePage extends AbstractPage {
+	
+	@FindBy(xpath = "//nav[@id = 'narvbarx']")
+	private NavigateMenu navigateMenu;
+	
+	@FindBy(xpath = "//*[@id = 'tbodyid']")
+	private List<Product> products;
+
+	//constructor
+	public HomePage(WebDriver driver) {
+		super(driver);
+		setPageAbsoluteURL(R.CONFIG.get(Configuration.Parameter.URL.getKey()));
+	}
+	
+	public ProductPage openProduct(String productString) {
+		for (Product product : products) {
+            String currentProduct = product.getTitle();
```

Comment:
You have to create your own method getTitle() in Product.java
Currently it's using wrong method, it will not work

Reply:
I'm not sure how or where I have to import a code o something in order to carina/selenium could do a click.. I can search on internet, but if you can suggest me anything it could be better.